recipes/rabbitmq/f52bca4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/sbin/rabbitmq-defaults b/sbin/rabbitmq-defaults
+index b845f74..95dccce 100755
+--- a/sbin/rabbitmq-defaults
++++ b/sbin/rabbitmq-defaults
+@@ -16,7 +16,7 @@
+ ##
+ 
+ ### next line potentially updated in package install steps
+-SYS_PREFIX=${RABBITMQ_HOME}
++SYS_PREFIX=${PREFIX}
```

Comment:
I think it will be better to have an activation script that sets `RABBITMQ_HOME` env variable.

See https://github.com/conda-forge/openjdk-feedstock/blob/master/recipe/build.sh#L35-L38 and https://github.com/conda-forge/openjdk-feedstock/tree/master/recipe/scripts for an example.